test/linqmacro.jl

```diff
@@ -38,6 +36,7 @@ xlinq2 = @linq df  |>
     transform(y = 10 * :x)  |>
     groupby(:b) |>
     orderby(-mean(:x))  |>
+    groupby(:b) |>
```

Comment:
This sounds really suboptimal. Maybe change the tests to do something more logical? Same below.